config/samples/config_v1beta1_policy_rego.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: config.ratify.deislabs.io/v1beta1
+kind: Policy
+metadata:
+  name: "regopolicy"
+spec:
+  parameters:
+    passthroughEnabled: false
+    policy: |
+      package ratify.policy
+
+      default valid := false
+
+      valid {
+        not failed_verify(input)
+      }
+
+      failed_verify(reports) {
```

Comment:
is it possible to add a comment in this yaml file to describe if this rego is "all artifact must be valid or "any"